lib/TWCManager/Vehicle/TeslaAPI.py

```diff
@@ -260,7 +260,7 @@ def apiRefresh(self):
             self.master.queue_background_task({"cmd": "saveSettings"})
 
     def car_api_available(
-        self, email=None, password=None, charge=None, applyLimit=None
+            self, email=None, password=None, charge=None, applyLimit=None
```

Comment:
Our indentation is performed automatically by ```black``` - so any changes will be lost next time we reblacken the code

Reply:
I'm sorry, you are right. That indentation change was by mistake.